src/Modules/Meetings/Domain/MeetingComments/Rules/CommentCanBeAddedOnlyByMeetingGroupMemberRule.cs

```diff
@@ -0,0 +1,22 @@
+﻿using CompanyName.MyMeetings.BuildingBlocks.Domain;
+using CompanyName.MyMeetings.Modules.Meetings.Domain.MeetingGroups;
+using CompanyName.MyMeetings.Modules.Meetings.Domain.Members;
+
+namespace CompanyName.MyMeetings.Modules.Meetings.Domain.MeetingComments.Rules
+{
+    public class CommentCanBeAddedOnlyByMeetingGroupMemberRule : IBusinessRule
```

Comment:
As far as i understand business rules on Miro comments can be added not only by meeting attendees but meeting group members in general.

Reply:
Yes, I think this is how it works in Meetup application.

Reply:
Yeah :) I decided to change behavior here after I've checked in Meetup.